.pipelines/templates/e2e-template.yaml

```diff
@@ -0,0 +1,51 @@
+jobs:
+  - job: E2E_Tests
+    pool:
+      name: nodesigtest-pool
+    timeoutInMinutes: 90
+    displayName: Run AgentBaker E2E
+    variables:
+      - group: ab-e2e
+    steps:
+      - bash: |
+          az login --identity
+          az account set -s $(AZURE_SUBSCRIPTION_ID)
+        displayName: az login
+      - bash: bash .pipelines/scripts/setup_go.sh
+        displayName: setup go
+      - bash: |
```

Comment:
should we give this step a `displayName` ?

Reply:
I copied most of the code from another pipeline.

It's confusing, but `displayName` are defined in the bottom of most steps